docs/book/vendor/install.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/sh
+# From https://github.com/japaric/trust/blob/2d08818370e618108b55fae9b243330640554dce/install.sh
```

Comment:
Is there a license for this file? Or was this created for this exact purpose?

Reply:
On the [master branch of the repo](https://github.com/japaric/trust) there's both an Apache and MIT license. Do you want that boilerplate-ified?

Reply:
They have a gh-pages branch that this file exists on normally, so it's served as a raw file.

Reply:
I'd defer to @detiber for license-y things 

I was mostly confused when opening the link, didn't find any license